README.md

```diff
@@ -1,40 +1,10 @@
-# regress - REGex in Rust with EcmaScript Syntax
+# regras3 - REGex in Rust with ActionScript 3 (~EcmaScript~) Syntax
```

Comment:
As there's no `s` at the end, I called it "REGex in Rust for ActionScript 3" in the repo description, isn't that better?

Reply:
have no strong feelings about this one way or the other.

Reply:
I would go for "REGex in Rust for ActionScript 3" as it makes more sense with the name